src/module/fluid.js

```diff
@@ -94,12 +94,20 @@ fluid.renderLoggingArg = function (arg) {
     return togo;
 };
 
-// Monkey-patch the built-in fluid.doLog utility to improve its behaviour within node.js - see FLUID-5475
-fluid.doLog = function (args) {
-    args = fluid.transform(args, fluid.renderLoggingArg);
+// Improve the behaviour of onkey-patch the built-in fluid.doLog utility to improve its behaviour within node.js - see FLUID-5475
```

Comment:
looks like a typo "onkey-patch" but wording in general seems to be a bit off; may be missing other words or changes.

Reply:
Sorry about this, some junk got inserted here as the result of a partial "undo" : P